features/git-prune-branches/feature_branch/behind_main/without_open_changes.feature

```diff
@@ -6,24 +6,44 @@ Feature: git prune-branches: remove stale feature branches when run on a feature
 
 
   Background:
-    Given I have a feature branch named "feature" behind main
-    And I have a feature branch named "stale_feature" behind main
-    And I am on the "feature" branch
+    Given the following commits exist in my repository
+      | BRANCH | LOCATION         | MESSAGE     | FILE NAME |
+      | main   | local and remote | main commit | main_file |
+    And I have a feature branch cut from "Initial commit" named "stale_feature_initial"
+    And I have a feature branch cut from "main commit" named "stale_feature_main"
```

Comment:
I think that should be a separate feature. This is `behind_main`. We should have an `even_with_main`.